### PR TITLE
FM2-507:FHIR IG Condition Resource

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/FM2-507
cc @pmanko  @mozzy11 
condition
conditionn
condtionnn

### CODE REVIEW

**Comment by sherrif10** [comment]
cc @pmanko

File: input/fsh/omrs-condition.fsh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Profile:      OMRSCondition
+Parent:       Condition
+Id:           omrs-Condition
+Title:        "OpenMRS condition"
+Description:  "A FHIR Condition as understood by OpenMRS"
+* identifier 0..*
```
Comment (pmanko) [inline input/fsh/omrs-condition.fsh:6]: I don't see an Identifier in the fhir2 translator. Also, I don't see support for `clinicalStatus` or `verificationStatus`

**Review by pmanko — CHANGES_REQUESTED** [review]

File: input/fsh/omrs-condition.fsh (hunk 1 above)
Comment (sherrif10) [reply input/fsh/omrs-condition.fsh]: Resolved Thanks.

**Comment by sherrif10** [comment]
search
Added search parameters.

**Pull request merged**